hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1967,10 +1967,12 @@
       name: chip_sw_csrng_fuse_en_sw_app_read
       desc: '''Verify the fuse input to CSRNG.
 
-            - Initialize the OTP with this fuse bit set to 1.
+            - Initialize the OTP with `HW_CFG.OTP_CTRL_PARAM_EN_CSRNG_SW_APP_READ_OFFSET` fuse bit 
```

Comment:
It might be more useful to write a sentence about this fuse bit rather than use the name used in the code. For example:
the fuse bit that controls whether software can read the CSRNG state.

Reply:
done